ios/Podfile.lock

```diff
@@ -336,23 +336,23 @@ PODS:
     - React
   - RNCMaskedView (0.1.10):
     - React
-  - RNCPushNotificationIOS (1.4.1):
+  - RNCPushNotificationIOS (1.5.0):
     - React
   - RNDeviceInfo (5.6.5):
     - React
-  - RNFBApp (8.4.1):
+  - RNFBApp (8.4.3):
     - Firebase/CoreOnly (~> 6.30.0)
-    - React
-  - RNFBCrashlytics (8.4.2):
+    - React-Core
+  - RNFBCrashlytics (8.4.5):
```

Comment:
Wasn't this removed? Is there a reason the pods are still included?

Reply:
I wasn't aware I just saw a flag on false, I won't touch it further